MANIFEST.in

```diff
@@ -27,42 +27,5 @@ recursive-include mne/html *.css
 
 recursive-include mne/io/artemis123/resources *
 
-recursive-include mne mne/datasets *.csv
-include mne/io/edf/gdf_encodes.txt
+recursive-include mne *.csv
 include mne/datasets/sleep_physionet/SHA1SUMS
-
-### Exclude
-
-recursive-exclude examples/MNE-sample-data *
-recursive-exclude examples/MNE-testing-data *
-recursive-exclude examples/MNE-spm-face *
-recursive-exclude examples/MNE-somato-data *
-exclude Makefile
-exclude .coveragerc
-exclude *.yml
-exclude ignore_words.txt
-exclude .mailmap
-recursive-exclude mne *.pyc
-
-recursive-exclude doc *
-recursive-exclude logo *
-
-exclude CODE_OF_CONDUCT.md
-exclude .github
-exclude .github/CONTRIBUTING.md
-exclude .github/ISSUE_TEMPLATE
-exclude .github/ISSUE_TEMPLATE/blank.md
-exclude .github/ISSUE_TEMPLATE/bug_report.md
-exclude .github/ISSUE_TEMPLATE/feature_request.md
-exclude .github/PULL_REQUEST_TEMPLATE.md
-
-# Test files
-
-recursive-exclude mne/io/tests/data *
-recursive-exclude mne/io/bti/tests/data *
-recursive-exclude mne/io/edf/tests/data *
-recursive-exclude mne/io/kit/tests/data *
-recursive-exclude mne/io/brainvision/tests/data *
-recursive-exclude mne/io/egi/tests/data *
-recursive-exclude mne/io/nicolet/tests/data *
-recursive-exclude mne/preprocessing/tests/data *
```

Comment:
I am not sure we want to do this. The reason it's there is to limit the size of the package on pypi.

Reply:
I get the same size `.tar.gz` either way, I'm guessing that the logic for what to include and exclude changed at some point. Hence the warnings above on `master` saying basically "these exclude lines do nothing".

Reply:
... well almost the same -- on `master`:
```
$ make clean
...
$ make wheel
...
$ ls -al dist/*.tar.gz
-rw-r--r-- 1 larsoner larsoner 6406781 Sep 17 14:09 dist/mne-0.19.dev0.tar.gz
```
This PR:
```
$ ls -al dist/*.tar.gz
-rw-r--r-- 1 larsoner larsoner 6406656 Sep 17 14:08 dist/mne-0.19.dev0.tar.gz
```
... but some shrinkage is to be expected because `MANIFEST.in` is in the `.tar.gz`, and this PR makes it smaller.

Reply:
I'm putting it back because `check-manifest` does not like these changes. It seems like a deficiency in `check-manifest` but we can just wait for them to fix it someday